test/e2e/common/node/runtimeclass.go

```diff
@@ -56,9 +56,11 @@ var _ = SIGDescribe("RuntimeClass", func() {
 	*/
 	framework.ConformanceIt("should reject a Pod requesting a non-existent RuntimeClass", f.WithNodeConformance(), func(ctx context.Context) {
 		rcName := f.Namespace.Name + "-nonexistent"
-		expectPodRejection(ctx, f, e2eruntimeclass.NewRuntimeClassPod(rcName))
-	})
+		err := expectPodRejection(ctx, f, rcName)
+		gotForbiddenError := apierrors.IsForbidden(err)
+		gomega.Expect(gotForbiddenError).To(gomega.BeTrueBecause("we expect a forbidden error"))
```

Comment:
See comment on `expectPodRejection`. I'd expect this to just be `framework.ExpectNoError(expectPodRejection(...`

Reply:
It returns an error though.

Reply:
Okay. I will return nil for only rejection error.

Reply:
updated.